docs/integrations-and-plugins/deltalake_versioning.md

```diff
@@ -12,7 +12,7 @@ In this example, you will use the `spaceflights-pandas` starter project which ha
 kedro new --starter spaceflights-pandas
 ```
 
-Kedro offers various connectors in the `kedro-datasets` package to interact with Delta tables: [`pandas.DeltaTableDataset`](https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-6.0.0/api/kedro_datasets.pandas.DeltaTableDataset.html), [`spark.DeltaTableDataset`](https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-6.0.0/api/kedro_datasets.spark.DeltaTableDataset.html), [`spark.SparkDataset`](https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-6.0.0/api/kedro_datasets.spark.SparkDataset.html), [`databricks.ManagedTableDataset`](https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-6.0.0/api/kedro_datasets.databricks.ManagedTableDataset.html), and [`ibis.FileDataset`](https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-6.0.0/api/kedro_datasets.ibis.FileDataset.html) support the delta table format. In this tutorial, we will use the `pandas.DeltaTableDataset` connector to interact with Delta tables using Pandas DataFrames. To install `kedro-datasets` alongwith dependencies required for Delta Lake, add the following line to your `requirements.txt`:
+Kedro offers various connectors in the `kedro-datasets` package to interact with Delta tables: [`pandas.DeltaTableDataset`](https://docs.kedro.org/projects/kedro-datasets/en/stable/api/kedro_datasets.pandas.DeltaTableDataset.html), [`spark.DeltaTableDataset`](https://docs.kedro.org/projects/kedro-datasets/en/stable/api/kedro_datasets.spark.DeltaTableDataset.html), [`spark.SparkDataset`](https://docs.kedro.org/projects/kedro-datasets/en/stable/api/kedro_datasets.spark.SparkDataset.html), [`databricks.ManagedTableDataset`](https://docs.kedro.org/projects/kedro-datasets/en/stable/api/kedro_datasets.databricks.ManagedTableDataset.html), and [`ibis.FileDataset`](https://docs.kedro.org/projects/kedro-datasets/en/stable/api/kedro_datasets.ibis.FileDataset.html) support the delta table format. In this tutorial, we will use the `pandas.DeltaTableDataset` connector to interact with Delta tables using Pandas DataFrames. To install `kedro-datasets` alongwith dependencies required for Delta Lake, add the following line to your `requirements.txt`:
```

Comment:
For the record, originally it was me suggesting the use of hardcoded versions in these docs https://github.com/kedro-org/kedro/pull/4483#discussion_r1967476408 but in retrospect, even if the risk of breakage slightly increases, this is just the right thing to do ✔️